src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackEditCopyUiTest.java

```diff
@@ -0,0 +1,93 @@
+package teammates.test.cases.ui.browsertests;
+
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.DataBundle;
+import teammates.common.util.Const;
+import teammates.common.util.ThreadHelper;
+import teammates.common.util.Url;
+import teammates.test.pageobjects.Browser;
+import teammates.test.pageobjects.BrowserPool;
+import teammates.test.pageobjects.InstructorFeedbackEditPage;
+import teammates.test.util.Priority;
+
+
+@Priority(-1)
```

Comment:
Is this needed? We should omit Priority for tests unless there are concurrency issues with other tests.